src/deluge/processing/sound/sound.cpp

```diff
@@ -3690,6 +3709,8 @@ void Sound::writeParamsToFile(ParamManager* paramManager, bool writeAutomation)
 	UnpatchedParamSet* unpatchedParams = paramManager->getUnpatchedParamSet();
 
 	unpatchedParams->writeParamAsAttribute("arpeggiatorGate", params::UNPATCHED_ARP_GATE, writeAutomation);
+	unpatchedParams->writeParamAsAttribute("arpeggiatorRatchetProbability", params::UNPATCHED_ARP_RATCHET_PROBABILITY,
```

Comment:
Sound checks ratchetProbability to read this parameter

You can put the UnpatchedSound enum into the template enum to string mapper class to avoid this error - https://github.com/SynthstromAudible/DelugeFirmware/blob/community/src/deluge/util/container/enum_to_string_map.hpp

Example here - https://github.com/SynthstromAudible/DelugeFirmware/blob/community/src/deluge/model/mod_controllable/filters/filter_config.cpp

Reply:
Fixed!